### PR TITLE
HTML log assets' filepaths must be relative to --assets option

### PR DESCRIPTION
**Pull request by happz**

### CODE REVIEW

File: gluetool/html_log.py [hunk 1]
```diff
@@ -295,18 +295,19 @@ def _snippet_filter(ctx, filepath, lineno, syntax, window=10):
...
 
 
-@jinja2.evalcontextfilter  # type: ignore
+@jinja2.contextfilter  # type: ignore
 def file_content_filter(ctx, value):
     # type: (Any, str) -> Union[str, Markup]
     # pylint: disable=unused-argument
     """
-    Return content of the given file.
+    Return content of the given file. File path imust be relative to an assets directory
```
Comment (thrix) [inline gluetool/html_log.py]: typo: imust

**Review by thrix — DISMISSED** [review]
Just small typo, otherwise LGTM

File: gluetool/html_log.py (hunk 1 above)
Comment (happz) [reply gluetool/html_log.py]: Ack, fixed.

**Review by thrix — APPROVED** [review]
LGTM

**Review by optak — APPROVED** [review]
LGTM

**Pull request merged**